rts/lra/trip-controller/src/main/java/io/narayana/rts/lra/demo/tripcontroller/TripController.java

```diff
@@ -220,5 +223,24 @@ private Booking bookFlight(String flightNumber, String bookingId) throws Booking
 
         return response.readEntity(Booking.class);
     }
+
+    @PUT
+    @Path("/after")
+    @Produces(MediaType.APPLICATION_JSON)
+    @AfterLRA
+    public Response afterEnd(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId, LRAStatus status) throws JsonProcessingException {
```

Comment:
I would like to point here as this is a quickstart which should show for the users how to use the spec.
I consider this complicated switch misleading for the quickstart use.

Reply:
@ochaloup  So, shall I remove the switch case

Reply:
```suggestion
    public Response afterEnd(@HeaderParam(LRA_HTTP_ENDED_CONTEXT_HEADER) URI lraId, LRAStatus status) throws JsonProcessingException {
```

Reply:
@mayankkunwar if I'm thinking from the perspective of need of the switch handling in the quickstart example, as it seems to me it does not bring any value for the developer of the method, then I think it's not necessary to be here
As it was already approved and merged then it's fine. I would just have a generic point that's good to trace down the context for the change, what is good for and what benefit it brings for the code around.

Reply:
Okay, Thanks